app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= link_to user_path(user), class: "user-list-item" do %>
+    <li class="list_item border-black ">
+      <%= image_tag user.photo %>
+      <div class="user_info">
+        <h2><%= user.name %></h2>
+        <p>Number of posts: <%= user.posts_counter %></p>
+      </div>
```

Comment:
- [OPTIONAL] For the post_counter to reflect the actual number of posts, I would advise you to use the `increment!` method instead of the `increment` method. 👍